app/views/developers/index.html.erb

```diff
@@ -46,9 +46,19 @@ div.desc {
 <body>
   <div class="container about-container">
     <div class="app-description">
-      <p>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Elit ullamcorper dignissim cras tincidunt lobortis. Sem et tortor consequat id porta. Imperdiet massa tincidunt nunc pulvinar sapien et ligula ullamcorper. Nec nam aliquam sem et tortor. Integer vitae justo eget magna fermentum iaculis. Id diam maecenas ultricies mi eget mauris pharetra et. Hendrerit gravida rutrum quisque non tellus orci. Vitae purus faucibus ornare suspendisse sed nisi lacus sed. Imperdiet nulla malesuada pellentesque elit. In pellentesque massa placerat duis ultricies lacus sed turpis tincidunt. Faucibus in ornare quam viverra orci sagittis eu volutpat. Non tellus orci ac auctor. In iaculis nunc sed augue lacus viverra. Varius quam quisque id diam vel quam elementum pulvinar etiam.
-      </p>
+    <br>
+      <p>We are the team behind Podsmack. A user friendly web application focused around assisting in finding your next favorite podcast.</p>
+      <p>The goal of this project was to create a successful Rails application from an idea to delivery within 10 days, focused around solving a problem. The podcast world is full of options, so how do you narrow down the search based on your interests? What we have created authenticates through a third-party service (Google), and consumes an api (Listen Notes) to find podcasts based around selecting location, custom tags, content rating, or searching through name.</p>
+      <p> We used an agile process developing our features. Also, Configured a continuous integration server, caching, and implemented a Sinatra Microservice. The tech stack used for this project are the following:</p>
+        <ul>
+          <li>Ruby on Rails</li>
+          <li>Active Record</li>
+          <li>Sinatra</li>
+          <li>Travis CI</li>
+          <li>Bootstrap</li>
```

Comment:
We are pros now

Reply:
YES WE ARE
![](https://media.giphy.com/media/LPrHgad7ETclrgwjge/giphy.gif)